handlers/desired_lrp_handlers.go

```diff
@@ -489,3 +505,32 @@ func (h *DesiredLRPHandler) updateInstances(ctx context.Context, logger lager.Lo
 		}
 	}
 }
+
+func (h *DesiredLRPHandler) logDesiredLrpParsingErrors(err *models.Error, processGuid string) error {
+	if err == nil {
+		return errors.New("parsing error is nil")
+	}
+
+	appGuid := parseAppGuidFromProcessGuid(processGuid)
+	if appGuid == "" {
+		return errors.New("app guid is empty")
+	}
+
+	tags := map[string]string{
+		"source_id": appGuid,
+	}
+
+	return h.metronClient.SendAppErrorLog(
+		fmt.Sprintf("Error parsing request for app with guid %s, %s, %s", appGuid, err.GetType(), err.GetMessage()),
+		BbsLogSource,
+		tags)
+}
+
+func parseAppGuidFromProcessGuid(processGuid string) string {
+	parts := strings.SplitN(processGuid, "-", 6)
+	if len(parts) < 5 {
```

Comment:
**Issue**

Can you explain what these numbers mean with a comment or a variable/constant? Without sufficient context, these numbers are magic numbers.